pkg/api/accesscontrol.go

```diff
@@ -606,14 +606,41 @@ func (hs *HTTPServer) declareFixedRoles() error {
 		Grants: []string{"Admin"},
 	}
 
+	snapshotsCreateRole := ac.RoleRegistration{
+		Role: ac.RoleDTO{
+			Name:        "fixed:snapshots:create",
+			DisplayName: "Create",
+			Description: "Create snapshots",
+			Group:       "Snapshots",
+			Permissions: []ac.Permission{
+				{Action: dashboards.ActionSnapshotsCreate, Scope: dashboards.ScopeDashboardsAll},
```

Comment:
Are the creation and deletion actions scoped to a dashboard or not? This makes me think that they are, but the doc changes and the checks below are not including a scope.

Reply:
The relationship between a snapshot and a dashboard is not mandatory; a snapshot could exist after the original dashboard is deleted, so they are independent. Would it make sense to scope it to dashboards?

Reply:
I want to scope snapshot creation to a dashboard uid and check for reading permissions.